qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/settings/QuteValidationTypeSettings.java

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************************
```

Comment:
This QuteValidationTypeSettings is not used in this PR?

Reply:
I didnt see an existing one and I branched from master, is there another `QuteValidationTypeSettings`?

Reply:
I don't understand the PR. Where QuteValidationTypeSettings is used?

In QuteDiagnostics you should use the severity coming from QuteValidationTypeSettings for undefied object when diagnostic for UndefinedObject is created https://github.com/redhat-developer/quarkus-ls/blob/b247849e376493117f3ab704d9f3060003ce86f0/qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteDiagnostics.java#L544